natgeo/mortar
# SYTO-885 + SYTO-886: Create and Style "Most Read" component and style containing package

https://jira.natgeo.com/browse/SYTO-885
https://jira.natgeo.com/browse/SYTO-886

These commits create the "Most Read" component and finishes up styling for the content package containing it and "The Latest"

To test:

- Pull feature/SYTO-885
- Run `npm test`
- Verify tests and linting pass
- Run `gulp serve`
- Navigate over to: http://localhost:3000/contentPackages/ and/or http://localhost:3000/contentPackages/contentPackage/index.html
- Verify layout
